tools/config/go-templates/model_simple.mustache

```diff
@@ -17,6 +17,12 @@ type {{classname}} struct {
 {{#description}}
 	// {{{.}}}
 {{/description}}
+{{#isReadOnly}}
+  // Read only field
```

Comment:
in go you should always end your docs with a full stop (.), also there's something odd with the spacing compared to the examples above

Also godoc thing, new lines not necessarily translate to new lines when rendered in https://pkg.go.dev/go.mongodb.org/atlas-sdk@v1.0.0/admin I would evaluate if you really want it as a new line or should be inline

Reply:
Added the full stop and fixed the indentation.

Q on the other point: do you mean that it would result in something like:
```
// Some description. // Write only field.
```

Reply:
> do you mean that it would result in something like

More like
> Some description. Write only field.

when rendered there's no comment markers

Reply:
I think it's fine with it rendering on the same line 👍

Reply:
works for me if it works for you